src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+
+const averageLifespan = (people) => people
+  .reduce((sum, element) =>
+    sum + (element.died - element.born), 0) / people.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menLifespan = people
```

Comment:
Це список чоловіків, а не список їх тривалості життя
```suggestion
  const men = people
```